sys/dev/netmap/netmap.c

```diff
@@ -2117,6 +2117,54 @@ netmap_csb_validate(struct netmap_priv_d *priv, struct nmreq_opt_csb *csbo)
 	return 0;
 }
 
+/* Validate the na can accomodate the given MTU.
+ * @return EINVAL if the NA cannot be set to mtu
+ *         0 if it can.
+ */
+int
+netmap_mtu_validate(const struct netmap_adapter *na, unsigned mtu) {
```

Comment:
This is more `netmap_buf_size_validate`, because that's what we are validating (MTU is given).